src/astrid_sedal_ex/ex04/myrand.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Astrid Hæve Sedal'
+__email__ = 'astrised@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.r_n = seed
+        self.a = 7**5
+        self.m = 2**31 - 1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime `rand()` is called